statbot/sql.py

```diff
@@ -1002,11 +1138,10 @@ def remove_old_members(self, trans, guild):
                 ))
         result = trans.execute(sel)
 
-        for row in result.fetchall():
-            user_id = row[0]
+        for user_id, _, _, _, _ in result.fetchall():
```

Comment:
wouldn't it be possible to just fetch the single column instead of this?

Reply:
I can't find a way to do this.

Reply:
http://docs.sqlalchemy.org/en/latest/core/selectable.html#sqlalchemy.sql.expression.Select.with_only_columns

Reply:
Alright.